htdocs/societe/card.php

```diff
@@ -128,12 +136,13 @@
 	$objcanvas->getCanvas('thirdparty', 'card', $canvas);
 }
 
-$permissiontoread = $user->rights->societe->lire;
-$permissiontoadd = $user->rights->societe->creer; // Used by the include of actions_addupdatedelete.inc.php and actions_lineupdown.inc.php
+// Permissions
+$permissiontoread 	= $user->rights->societe->lire;
+$permissiontoadd 	= $user->rights->societe->creer; // Used by the include of actions_addupdatedelete.inc.php and actions_lineupdown.inc.php
 $permissiontodelete = $user->rights->societe->supprimer || ($permissiontoadd && isset($object->status) && $object->status == 0);
-$permissionnote = $user->rights->societe->creer; // Used by the include of actions_setnotes.inc.php
-$permissiondellink = $user->rights->societe->creer; // Used by the include of actions_dellink.inc.php
-$upload_dir = $conf->societe->multidir_output[isset($object->entity) ? $object->entity : 1];
+$permissionnote 	= $user->rights->societe->creer; // Used by the include of actions_setnotes.inc.php
```

Comment:
Note that adding a space is useless because tool to sanitize and standardize code and a lot of IDE remove them to keep only 1 space. So this change will probably be reverted by another PR done by someone else.